tests/api/api/test_background_tasks.py

```diff
@@ -1,21 +1,26 @@
+import datetime
 import http
 import typing
 import unittest.mock
 
 import fastapi
 import fastapi.testclient
 import pytest
+import requests
 import sqlalchemy.orm
 
 import mlrun.api.api.deps
 import mlrun.api.main
 import mlrun.api.schemas
 import mlrun.api.utils.auth.verifier
 import mlrun.api.utils.background_tasks
+import mlrun.api.utils.clients.chief
 
 test_router = fastapi.APIRouter()
 
 
+# because we don't have specific endpoints for creating background tasks, but only through start_function /
+# operations/migrations, with the routes bellow we simulate those actions that would be performed
```

Comment:
That's not the reason, to overcome what you said we could simply call the handler itself, the reason we can't do that, and have to declare an endpoint is that our class is built on top of FastAPI's background_tasks mechanism, and to get this class, we must trigger an endpoint